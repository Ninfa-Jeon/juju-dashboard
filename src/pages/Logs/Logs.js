import React from "react";

import Layout from "components/Layout/Layout";

export default function Logs() {
  return (
    <Layout>
      <div className="p-strip">
        <div className="row">
          <h2>Logs</h2>
        </div>
      </div>
    </Layout>
  );
}
