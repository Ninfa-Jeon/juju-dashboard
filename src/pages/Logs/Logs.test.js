import React from "react";
import { shallow } from "enzyme";
import Logs from "./Logs";

describe("Logs page", () => {
  it("renders without crashing and matches snapshot", () => {
    const wrapper = shallow(<Logs />);
    expect(wrapper).toMatchSnapshot();
  });
});
