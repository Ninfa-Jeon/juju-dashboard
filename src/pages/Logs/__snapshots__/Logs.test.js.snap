// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logs page renders without crashing and matches snapshot 1`] = `
<Layout>
  <div
    className="p-strip"
  >
    <div
      className="row"
    >
      <h2>
        Logs
      </h2>
    </div>
  </div>
</Layout>
`;
